packages/paste-core/components/display-pill-group/src/DisplayPill.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxElementProps} from '@twilio-paste/box';
+import {secureExternalLink} from '@twilio-paste/anchor';
+
+interface DisplayPillProps extends BoxElementProps {
+  href?: string;
+  element?: string;
+  children: React.ReactNode;
+}
+
+export const DisplayPill = React.forwardRef<HTMLElement, DisplayPillProps>(
+  ({element = 'DISPLAY_PILL', ...props}, ref) => {
+    return (
+      <Box as="li" listStyleType="none">
```

Comment:
What does the `listStyleType` determine? I haven't used it before.

Reply:
Browsers have default CSS, so things like `li` have bullets by default. `list-style-type` is CSS to change that behavior: https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type